src/lib.rs

```diff
@@ -235,7 +235,7 @@ impl BitFlags {
 }
 
 pub mod channel;
-pub use channel::{ChannelId, ChannelSelection};
+pub use channel::ChannelId;
```

Comment:
I do not think we should expose `ChannelSelection` publicly.

Reply:
It isn't?